README.md

```diff
@@ -90,25 +89,25 @@ To test out your generated bitstream, you can upload it to a emulated FPGA devic
 
 ## Documentation
 
-* [Logic User Guide]()
+* [Logik User Guide]()
 * [Logik Reference Manual]()
 * [SiliconCompiler Documentation](https://docs.siliconcompiler.com/en/stable/)
 
 
 ## Installation
 
-Logik is available as wheel packages on PyPI for macOS, Windows and Linux platforms. For working Python 3.8-3.12 environment, just use pip to install.
+Logik is available as wheel packages on PyPI for macOS, Windows and Linux platforms. For a Python 3.8-3.12 environment, just use pip to install.
 
 ```sh
 python -m pip install --upgrade logik
 ```
 
-Running natively on your local machine will require a number of prerequisite tools (Yosys, Surelog, VPR) a Automated Ubuntu based install scripts are included for convenience within the SiliconCompiler project. Detailed instructions for installing all tools can be found in the [SiliconCompiler Installation Guide](https://docs.siliconcompiler.com/en/stable/user_guide/installation.html#external-tools).
+Running natively on your local machine will require a number of prerequisite tools (Yosys, Surelog, VPR). Automated Ubuntu based install scripts are included for convenience within the SiliconCompiler project. Detailed instructions for installing all tools can be found in the [SiliconCompiler Installation Guide](https://docs.siliconcompiler.com/en/stable/user_guide/installation.html#external-tools).
 
 
 ## Running Docker
 
-A docker container is available for folks who prefer that route.
+A [Docker image](https://github.com/siliconcompiler/siliconcompiler/pkgs/container/sc_runner) with the required tools is available for folks who prefer that route. The following command starts a new container from that image and maps the local directory `sc_work` to the path `/sc_work` in the container.
```

Comment:
@volkamir had this feedback, and I concur:  "The wording "A docker container is available for folks who prefer that route." seems too "loose" to me. I would find it more professional if it was something like "A docker container is provided for users who wish to avoid the installation of the pre-requisite tools"

Reply:
sounds good, will update that